packages-native/maps/README.md

```diff
@@ -1,3 +1,56 @@
-# Maps widget
+## Maps widget
 
-Show locations on an interactive map.
+Use maps widget to show list of locations on an interactive map in your native apps.
+
+### Using in Native Builder Generated Apps
```

Comment:
This should also be explained for iOS

Reply:
I would not spend time on it ATM. Lets improve our README step by step